frontend/js/src/brainzplayer/BrainzPlayer.tsx

```diff
@@ -592,7 +591,7 @@ export default class BrainzPlayer extends React.Component<
         createNotification(title, artist, album, artwork?.[0]?.src);
       } else {
         const message = (
-          <>
+          <div className="alert">
```

Comment:
As described above, I suggest:
```suggestion
          <div className="brainzplayer-alert">
```

Reply:
Yes, I think "brainzplayer-alert" is better. I will try to avoid using confusing class names in the future.